web/src/settings_invites.ts

```diff
@@ -160,10 +160,11 @@ function do_revoke_invite({
         blueslip.error("Invite revoking canceled due to non-matching fields.");
         ui_report.client_error(
             $t_html({
-                defaultMessage: "Resending encountered an error. Please reload and try again.",
+                defaultMessage: "Error: Could not revoke invitation.",
             }),
-            $("#home-error"),
+            $("#revoke_invite_modal #dialog_error"),
```

Comment:
Loading spinner is shown even after showing this error. It needs to be hidden.

Reply:
I apologize for the inconvenience caused. I will update this in the next push, waiting for a final decision on the disabled state of the Resend button.